07-asymmauth.tex

```diff
@@ -191,7 +191,7 @@ \subsection{Schlüssel- und Signaturerzeugung}
 Zahl $e \in \{1, \dots, p - 1\}$ gewählt,
 wobei $p=|\G|$. Damit berechnet man 
 \begin{align*}
-a &:= g^e \in \G\\
+a &:= g^e mod p \in \G\\
```

Comment:
p=|G| ist hier die Ordnung der Gruppe. Allerdings müsste man hier modulo die Primzahl rechnen, oder nicht?
Bzw p ist hier doppeldeutig, da einmal in Z_p verwendet und dann als Gruppenordnung redefiniert.